_pages/index.markdown

```diff
@@ -35,7 +35,9 @@ feature_row:
 
 {% include feature_row %}
 
-Orbital uses Mockdefinitions to match incoming requests. A Mockdefinition is an annotated OpenAPI spec (written in JSON) that contains the scenarios used to match requests for each endpoint. The OpenAPI spec defines a standard, programming language-agnostic interface description for APIs, allowing humans and computers to understand the capabilities of a service without source code.
+Orbital is an HTTP mocking tool that enables enables rapid _microservice_ development and testing. Orbital supports mocking services by using the OpenAPI spec in both YAML and JSON. Orbital allows for parallel development between front-end and back-end teams by reducing the dependencies between each other. This allows for increased productivity because the back-end team does not have to wait for the front-end team, and vice-versa. Deploying Orbital in your organization is very easy, and can remove these dependencies.
```

Comment:
can we remove ```HTTP``` from ```Orbital is an HTTP mocking tool``` and I think we can replace ```an``` for ```a``` too

Reply:
Done and done. Removed from the subtitle as well.